identity-verification/module_1/1.0 SETUP - Gather images and setup S3 buckets for IDV modules.ipynb

```diff
@@ -87,10 +87,11 @@
    "source": [
     "s3 = boto3.client('s3')\n",
     "try:\n",
-    "   s3.create_bucket(\n",
-    "       Bucket=bucket_name,\n",
-    "       CreateBucketConfiguration={'LocationConstraint': aws_region}\n",
-    "   )\n",
+    "    if aws_region == \"us-east-1\":\n",
+    "        s3.create_bucket(Bucket=bucket_name)\n",
+    "    else:   \n",
+    "        s3.create_bucket(Bucket=bucket_name,\n",
+    "           CreateBucketConfiguration={'LocationConstraint': aws_region})\n",
```

Comment:
```suggestion
    "    s3.create_bucket(Bucket=bucket_name)\n",
```

I understand the issue but I still think this should be sufficient? If the client is initialised in a given region, the bucket creation will be in that region. This should work on all regions.
(https://github.com/boto/boto3/issues/125)